libs/isograph-compiler/isograph-config-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ConfigFile",
+  "type": "object",
+  "required": [
+    "project_root",
+    "schema"
+  ],
+  "properties": {
+    "$schema": {
```

Comment:
lol is this a typo or is schema a special field?

Reply:
Yes, this is intentional